pages/en/lb4/Schemas.md

```diff
@@ -144,6 +144,76 @@ class Customer extends Entity {
 }
 ```
 
+### JSON Schema inferrence
+With type information inferred from using `@model` and `@property` decorators,
+we've created a module to easily build a JSON Schema from a decorated model.
+In the `@loopback/repository-json-schema` module, we provide `getJsonSchema`
+which accesses the metadata stored by the decorators to build a matching
+JSON Schema of your model.
+
```

Comment:
Suggested rewording: 
Use the `@loopback/repository-json-schema` module to build a JSON schema from a decorated model. Type information is inferred from the `@model` and `@property` decorators. The `@loopback/repository-json-schema` module contains the `getJsonSchema` function to access the metadata stored by the decorators to build a matching JSON Schema of your model.